blocks/donor-wall/edit/inspector.js

```diff
@@ -2,145 +2,250 @@
+                                            label={ __( 'Read More Text', 'give' ) }
+                                            value={ readMoreText }
+                                            onChange={ ( value ) => saveSetting( 'readMoreText', value ) } />
+                                    </>
+                                }
+                    </PanelBody>
+                </Panel>
+                <Panel>
+                    <PanelBody title= {__('Wall Settings', 'give')} initialOpen={ true }>
+                        <SelectControl
+                            className="give-donor-wall-inspector"
+                            label={ __( 'Sort By', 'give' ) }
+                            name="orderBy"
+                            value={ orderBy }
+                            options={ giveDonorWallOptions.orderBy }
+                            onChange={ ( value ) => saveSetting( 'orderBy', value ) } />
+                        <SelectControl
+                            className="give-donor-wall-inspector"
+                            label={ __( 'Order', 'give' ) }
+                            name="order"
+                            value={ order }
+                            options={ giveDonorWallOptions.order }
+                            onChange={ ( value ) => saveSetting( 'order', value ) } />
+                        <SelectControl
+                            className="give-donor-wall-inspector"
+                            label={ __( 'Filter', 'give' ) }
+                            name="filter" value={ filter }
+                            options={ giveDonorWallOptions.filter }
+                            onChange={ ( value ) => saveSetting( 'filter', value ) } />
+
+                        {filterValue(filter)}
+                    </PanelBody>
+                </Panel>
+                <Panel>
+                    <PanelBody title= {__('Wall Interaction', 'give')} initialOpen={ true }>
+                        <TextControl
+                            className="give-donor-wall-inspector"
+                            name="donorsPerPage"
+                            label={ __( 'Donors Per Page', 'give' ) }
+                            value={ donorsPerPage }
+                            onChange={ ( value ) => saveSetting( 'donorsPerPage', value ) }
+                            help={ __('How many donors should show up on the initial page load?', 'give' ) }
+                        />
+                        <TextControl
+                            className="give-donor-wall-inspector"
+                            name="loadMoreText"
+                            label={ __( 'Load More Text', 'give' ) }
+                            value={ loadMoreText }
+                            onChange={ ( value ) => saveSetting( 'loadMoreText', value ) } />
+                    </PanelBody>
+                </Panel>
+            <PanelColorSettings
```

Comment:
I think using the `ColorPalette` component directly would be a more efficient UI (less clicks) by bringing the control inline with the inspector fields.

For comparison:

![image](https://user-images.githubusercontent.com/10858303/171901312-9f15557c-f58e-406c-946d-d1de75071178.png)

```js
import {ColorPalette} from '@wordpress/block-editor';
<Panel>
    <PanelBody title= {__('Color', 'give')} initialOpen={ true }>
        <ColorPalette
            value={color}
            onChange={( value ) => setAttributes( { color: value } )}
        />
    </PanelBody>
</Panel>
```